qa/tasks/cephfs/test_admin.py

```diff
@@ -1424,15 +1424,49 @@ def test_multiple_path_rw(self):
 
         self._remount_and_run_tests(FS_AUTH_CAPS, keyring)
 
+    def test_when_MDS_caps_needs_update_but_others_dont(self):
+        '''
+        Test that the command "ceph fs authorize" successfully updates MDS
+        caps even when MON and OSD caps don't need an update.
+
+        Tests: https://tracker.ceph.com/issues/64182
+
+        In this test we run -
+
+            ceph fs authorize cephfs2 client.x / rw
+            ceph fs authorize cephfs2 client.x /dir1 rw
+            ceph fs authorize cephfs2 client.x /dir2 rw
+
+        The first command will create the keyring by adding the MDS cap for
+        root path & MON and OSD caps with name of the FS name (say "cephfs1").
+        But second and third command don't need to add MON and OSD caps since
+        cap for "cephfs2" has been granted already. Thus, second and third
+        command only need to update the MDS cap, thus testing the bug
+        under consideration here.
+        '''
+        PERM = 'rw'
+        FS_AUTH_CAPS = (('/', PERM), ('dir1', PERM), ('/dir2', PERM))
```

Comment:
Should this be `/dir1` or maybe just "dir2", but not both.

Reply:
This was too ensure that adding multiple caps for first FS works, which was seen during investigating this bug.

Reply:
Sure, but I'm pointing to `dir1` and `/dir2` (one with `/` and the other without it).

Reply:
Ah, I suppose you mean either to have `/dir1` and `/dir2` or to have `dir1` and `dir2`...

Reply:
Anyways, this part is not vital to the test case, so I've removed it/

Reply:
Done. Marking this thread resolved.